root/static/styles/favicons.less

```diff
@@ -29,6 +29,7 @@
 .favicon("beatport", 32);
 .favicon("bigcartel", 32);
 .favicon("bookbrainz", 32);
+.favicon("boomkat", 16);
```

Comment:
16 is the default so it does not need specifying (is there a reason you went for 16 though? we usually do 32 but probably fine either way)

Reply:
Strangely they only offer a 16px favicon and I couldn't find any other copies of the same icon linked anywhere in the page source:

https://boomkat.com/assets/favicon-eec454f2397be06df3606b27e5be871d3847fffbfdcc012af30e26439aec6ed4.ico

I've removed the default parameter value.